core/src/components/searchbar/searchbar.tsx

```diff
@@ -101,9 +115,14 @@ export class Searchbar implements ComponentInterface {
   @Prop() searchIcon = 'search';
 
   /**
-   * If `true`, show the cancel button.
+   * Sets the behavior for the cancel button.
+   * Setting to `true` shows the cancel button on focus.
+   * Setting to `false` disables the cancel button.
+   * Setting to `persist` shows the cancel button on focus
```

Comment:
```suggestion
   * Setting to `"persist"` shows the cancel button on focus
```

Reply:
GitHub won't let me directly commit these suggestions for some reason, so I've made them manually and pushed an update.